### PR TITLE
Starting Log modified

### LINKED ISSUE #81: Server starting log
When the server start it should log some information with time stamps.

It would be nice to have some ASCII art with a logo.

### PR DESCRIPTION
**Pull request by ZimGil**
Added some ASCII art at server start and made it a little nicer [resolves #81]

Also, installed [colors](https://www.npmjs.com/package/colors)

### CODE REVIEW

File: server/server.ts [hunk 1]
```diff
@@ -131,12 +131,28 @@ app.post('/DEVchangeDir', (req, res) => {
...
-  console.log(`Listening on port ${PORT}!`);
+  startingASCII();
+  // TODO: Add version/build number here
+  console.log('Listening on ', colors.grey(`http://localhost:${PORT}`));
   if (isOpenBrowser) {
     openBrowser(PORT);
   }
 });
 
+function startingASCII() {
```
Comment (ortichon) [inline server/server.ts]: move this function to a helper.
could be called something like `ascii-logo-printer.helper.ts`
then export and import it.

File: server/server.ts [hunk 2]
```diff
@@ -131,12 +131,28 @@ app.post('/DEVchangeDir', (req, res) => {
 
 app.listen(PORT, () => {
   console.clear();
-  console.log(`Listening on port ${PORT}!`);
+  startingASCII();
```
Comment (ortichon) [inline server/server.ts]: consider changing the name of this function to something more descriptive.

File: server/logo-printer.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import colors = require('colors/safe');
+
+export class LogoPrinter {
```
Comment (ortichon) [inline server/logo-printer.ts]: does not have to be a class. could just be:
```typescript
export function printAsciiLogo() {
...
}
```

Comment (ZimGil) [reply server/logo-printer.ts]: cool

**Pull request merged**